server/scheduler/GeneticAlgorithm.py

```diff
@@ -592,12 +592,13 @@ def __str__(self):
 
 class Population:
     # Defining variables for Population of schedules.
-    def __init__(self, size):
+    def __init__(self, size, semYear):
```

Comment:
I'm not sure we need to pass the semester and year into the population and schedule classes.

Reply:
I believe that the response above answers this question